src/java.base/share/classes/java/lang/StringCoding.java

```diff
@@ -522,85 +48,18 @@ final String requestedCharsetName() {
      */
     private static native void err(String msg);
```

Comment:
A separate cleanup could remove this unused method and the corresponding native code in StringCoding.c.